src/controllers/donorController.ts

```diff
@@ -129,7 +125,7 @@ export const availPickup = (req: Request, res: Response) => {
         'availability.startTime': { $lte: new Date() },
         'availability.endTime': { $gte: new Date() }
     })
-        .populate('donor', '_id name latitude longitude')
+        .populate('user', '_id donorInfo.name donorInfo.address')
```

Comment:
Same as above, can you make this populate the longitude and latitude, in addition to the address?